### PR TITLE
BitGuardian.BitDriverUpdater version 1.0.0.7

### PR DESCRIPTION
**Pull request by Meteor24**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15812)

### CODE REVIEW

**Comment by KaranKad** [comment]
`PackageIdentifier` should be  `BitGuardian.BitDriverUpdater` instead of `BitGuardianGmbH.BitDriverUpdater`

**Comment by KaranKad** [comment]
Not just changing the pr title. You need to edit the .yaml file. If you need help, tell me I will submit this package for you.

**Comment by Meteor24** [comment]
if i have to edit the yaml file does that also mean that i have to resubmit the PR?

**Comment by KaranKad** [comment]
You can edit this existing pr. Make changes to the yaml file and folder name.

**Comment by Meteor24** [comment]
it got validated so what should i do now?

**Comment by Meteor24** [comment]
okay i took out the gmbh part in the file code but GmBH is still in the file name

**Comment by KaranKad** [comment]
You will also need to edit the folder name from `/BitGuardianGmbH/BitDriverUpdater/1.0.0.7/BitGuardianGmbH.BitDriverUpdater.installer.yaml` to `/BitGuardian/BitDriverUpdater/1.0.0.7/BitGuardian.BitDriverUpdater.installer.yaml` and for other yaml files as well.

File: manifests/b/BitGuardianGmbH/BitDriverUpdater/1.0.0.7/BitGuardianGmbH.BitDriverUpdater.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BitGuardian.BitDriverUpdater
+PackageVersion: 1.0.0.7
+PackageLocale: en-US
+Publisher: Bit Guardian GmbH
+PackageName: Bit Driver Updater
+License: Copyright © 2019 Bit Guardian GmbH
+ShortDescription: Bit Driver Updater Setup
```
Comment (ItzLevvie) [inline manifests/b/BitGuardianGmbH/BitDriverUpdater/1.0.0.7/BitGuardianGmbH.BitDriverUpdater.locale.en-US.yaml:9]:
```suggestion
ShortDescription: Bit Driver Updater helps computers to detect defective drivers, install driver updates when needed, and fix other driver-related issues.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Meteor24** [comment]
im just going to resubmit this whole thing

**Pull request closed without merging**